.dockerenvrc

```diff
@@ -1,4 +1,5 @@
-NODE_ENV='production'
```

Comment:
Any reason why env is changed to dev for Docker?

Reply:
should be production, will change back